modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/store/withStore.es.js

```diff
@@ -147,10 +169,23 @@ export default Component => {
 			};
 		}
 
+		getFormId() {
+			const form = this.getFormNode();
+
+			return form && form.dataset.ddmforminstanceid;
+		}
+
 		getFormNode() {
 			return dom.closest(this.element, 'form');
 		}
 
+		getFormTitle() {
+			const form = this.getFormNode();
+			const formName = form && form.querySelector('.ddm-form-name');
+
+			return formName && formName.textContent;
```

Comment:
@interaminense, is there any chance of it return a `false` value instead of `undefined`?

Reply:
Yes, it was possible to return `undefined`. But since I used this **getFormTitle** just to trigger the **formViewed** event, this event will no longer be fired in this withStore class, it will continue to fire in the `view.jsp` file, there is already the title. So I am removing this method in this commit: https://github.com/natocesarrego/liferay-portal/pull/177/commits/8e6f577daa6f357cdf693dcae56e444f0603d45f